tools/retis_in_container.sh

```diff
@@ -35,11 +35,19 @@ else
 	exit -1
 fi
 
+# Determine if OVS is installed on the host and, if so, mount its binary.
+if binary=$(command -v ovs-vswitchd); then
+    ovs_binary_mount="-v ${binary}:${binary}:ro"
+else
```

Comment:
two small nits:

- tab vs spaces
- I think the else branch could be safely removed

Reply:
Didn't notice we also use tabs in shell scripts.
Regarding the else branch... well ok I can remove it. I tend to use "set -eu" which will error if I don't but in this case I guess we can..

Reply:
saw your update, thanks. The second one (else) was of course perfectly fine either way. Just included the comment while at it.